lib/screens/navigation_screen.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:social_media/screens/NotificationsPage.dart';
+import 'package:social_media/screens/ProfilePage.dart';
+import 'package:social_media/screens/HomePage.dart';
+import 'package:social_media/screens/SearchPage.dart';
+
+class NavigationScreen extends StatefulWidget {
+  const NavigationScreen({Key? key}) : super(key: key);
+
+  @override
+  State<NavigationScreen> createState() => _NavigationScreenState();
+}
+
+class _NavigationScreenState extends State<NavigationScreen> {
+  int _currentIndex = 0;
+
+  final List<Widget> pages = [
+    HomePage(),
+    Scaffold(
```

Comment:
you don't need the scaffold it doesn't serve any purpose you can get rid of it and replace it with 
SearchPage(),
```suggestion
    SearchPage(),
```